### ISSUE TITLE
Calculation: fractional results for integrals and derivatives

### ISSUE THREAD

**Issue by XanderLeaDaren** [numworks/epsilon#1720]
#### Problem you'd like to fix
When calculating an integral or a derivative, the answer is displayed as a decimal, and there is no option to get any fractional form for it (nor improper nor mixed fraction).

#### Describe the solution you'd like
Example for an integral.

Left: current calculation / Right: proposition
![proposition_integral_fractional-result](https://user-images.githubusercontent.com/28948599/100107412-e4ccc180-2e69-11eb-8267-b6e75b4a779a.png)

#### Describe alternatives you've considered
None found.

#### Additional context and ideas for solution
* I noticed that from version 15.0.0 (actual beta version), when calculating diff(x², x, 1/3), you get 0.6666(truncated)7 and the ability (via the ··· menu) to get the mixed fraction 0 + 2/3. However it’s not possible yet on version 14.4.0 (actual web version). This seems to indicate epsilon is now (= beta version) able to give additional results formats for derivatives, including fractional result. −> feature almost solved for derivatives.
* Sadly, for integrals, nor version 15.0.0 nor version 14.4.0 allow for such a behaviour. My question/idea is: how is epsilon handling integrals and derivatives? Is is the same core calculation? If so, integral calculations could give additional results (**1st step forward the solution**)
* As a final implementation of this, since mixed fraction is available for derivatives in version 15.0.0, displaying the improper fraction 65/3 (in above example) should be possible. Right?

**Comment by RedGl0w** [comment]
>     * I noticed that from version 15.0.0 (actual beta version), when calculating diff(x², x, 1/3), you get 0.6666(truncated)7 and the ability (via the ··· menu) to get the mixed fraction 0 + 2/3. However it’s not possible yet on version 14.4.0 (actual web version). This seems to indicate epsilon is now (= beta version) able to give additional results formats for derivatives, including fractional result. −> feature almost solved for derivatives.

Indeed, the main reason of this change is that, I think, the whole derivation algorithm changed, allowing some "more precisely" approximation, and so they can know have the fraction as result, and not only a float (this can be seen by #1393 or the speed in which a derivated graph is done). It seems that the algorithm of integration hasn't been changed, so the main issue is that the algorithm isn't exact.

**Comment by artaxxx** [comment]
Yes, we need exact computation on integrals for that. And it's not easy to do. I don't know if/when we'll do it yet!

**Comment by Gemme10** [comment]
Sorry to revive this thread.

I too would appreciate having the option to see the answer of the calculation in fractional form.

It appears to me that the exact computation of the integrals is happening now, so can we have the option of viewing the answer as a fraction please?

Example:
![screenshot4](https://github.com/numworks/epsilon/assets/163826672/7b321f11-4b96-41c8-90e0-5c71a0715023)

The exact answer (-1/12) has been obtained by the calculator as I get 0 when I add 1/12.
![screenshot5](https://github.com/numworks/epsilon/assets/163826672/b87a4c26-d3d7-483c-a72c-4b8324a8fd5b)

Hence the calculator could have also shown the fractional answer.

**Comment by MatejTymes** [comment]
Hi I have the same problem. For example casio as well as ti calculator (without CAS) have no problem with calculating, that this:
![missing-frac-representation](https://github.com/user-attachments/assets/532f39e2-2ed5-4adc-9a39-06fc1dc9f82f)
![no-additional-representation](https://github.com/user-attachments/assets/755ae17f-936d-4580-b481-c1d4d1f5ffdf)
can be actually represented in fractional form:
![frac-representation](https://github.com/user-attachments/assets/e9a111ca-9061-4f1b-96e3-48b0c43c13e0)
![ti-84-plus-output](https://github.com/user-attachments/assets/342080ae-50a5-4ef8-af24-a1ebcf8c67e9)
